docs/profile-main.md

```diff
@@ -39,6 +41,15 @@ foobar.save();
 
 Register profile `type` with the `path` you provide.
 
+### profile.inherit
+
+  - `type` string - Current type
+  - `parent` string - The parent type
+
+Set up a type of inheritance relationship
+When the data cannot be found, it will query the data at the parent type
```

Comment:
Sets the parent of the given type. This allows you get profile value from your parent (even parent's parent profile) when the value not found in current profile.